hibernate/src/main/java/com/example/homework6/controllers/EmailController.java

```diff
@@ -0,0 +1,151 @@
+package com.example.homework6.controllers;
+
+import com.example.homework6.*;
+import com.example.homework6.repos.EmailRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.Optional;
+
+@Controller
+public class EmailController {
+
+    @Autowired
+    EmailRepo emailRepo;
+
+    @Autowired
+    EmailSender emailSender;
+
+    @Autowired
+    TaskSchedule taskSchedule;
+
+
+    @GetMapping(value = "/create")
```

Comment:
Create is action to change something. It should be HTTP POST/PUT rather than HTTP GET